CONTRIBUTING.md

```diff
@@ -56,6 +56,19 @@ codebase, however you can always check to see if the source code is compliant by
 npm run lint
 ```
 
+For linting the native iOS package, we are using [Swift lint](https://github.com/realm/SwiftLint). You need to install it on your machine using the following command:
+
+```bash
+brew install swiftlint
+```
+
+And then you can run it by calling it from JS using:
+
+```bash
+yarn lint:ios
```

Comment:
I am getting a lot of errors and warninigs.

```
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/AnimationViewManagerModule.swift:36:1: warning: Line Length Violation: Line should be 120 characters or less; currently it has 131 characters (line_length)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/PlatformColor.swift:19:9: error: Identifier Name Violation: Variable name 'r' should be between 3 and 40 characters long (identifier_name)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/PlatformColor.swift:19:9: error: Identifier Name Violation: Variable name 'g' should be between 3 and 40 characters long (identifier_name)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/PlatformColor.swift:19:9: error: Identifier Name Violation: Variable name 'b' should be between 3 and 40 characters long (identifier_name)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/PlatformColor.swift:19:9: error: Identifier Name Violation: Variable name 'a' should be between 3 and 40 characters long (identifier_name)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:101:59: error: Force Cast Violation: Force casts should be avoided (force_cast)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:102:58: error: Force Cast Violation: Force casts should be avoided (force_cast)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:309:74: error: Force Cast Violation: Force casts should be avoided (force_cast)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:311:98: error: Force Cast Violation: Force casts should be avoided (force_cast)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:277:1: warning: Line Length Violation: Line should be 120 characters or less; currently it has 130 characters (line_length)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:311:1: warning: Line Length Violation: Line should be 120 characters or less; currently it has 133 characters (line_length)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:359:1: warning: Line Length Violation: Line should be 120 characters or less; currently it has 143 characters (line_length)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:132:13: warning: No Fallthrough only Violation: Fallthroughs can only be used if the `case` contains at least one other statement (no_fallthrough_only)
/Users/matin/work/personal/lottie-react-native/packages/core/ios/LottieReactNative/ContainerView.swift:14:1: warning: Type Body Length Violation: Type body should span 250 lines or less excluding comments and whitespace: currently spans 278 lines (type_body_length)
```

Reply:
We can take two approaches here, refactor, or shutting off some of the rules:

1. Number of characters in lines? Maybe appropriate to refactor
2. Variable names length? I think we can disable
3. Force Cast? Generally a big No-No, but i think for exceptional cases, we can add // swiftlint:disable force_cast (We have opt-outs)
4. No fall through? Generally Swift tries to be a lot stricter with Switch cases than TS/JS, so we can refactor
5. Body length violation? Generally, yes, don't write too big of a Swift file, but this might be a bit too strict

What do you think about each case? I can disable things globally with a simple .swiftlint file in the iOS directory, works a lot like .eslint

Reply:
Let's add swift lint file and suppress some of them.